docs/source/rag.rst

```diff
@@ -36,3 +36,66 @@ and create files in the vector store. Here is an example script:
      file_id=file.id,
    )
    print("Created vector store file. ID=%s" % vfs.id)
+
+Once the files are added into vector store, you can run the completion request with the RAG model.
+
+.. code-block:: python
+
+   from openai import OpenAI
+
+   client = OpenAI(
+     base_url="<Base URL (e.g., http://localhost:8080/v1)>",
+     api_key="<API key secret>"
+   )
+
+   completion = client.chat.completions.create(
+     model="google-gemma-2b-it-q4",
+     messages=[
+       {"role": "user", "content": "What is k8s?"}
```

Comment:
Is it possible to change the question to something that uses the RAG? Something like "What is LLM Operator?".

Reply:
Any question is fine, as long as the files have related content